tock/hours/views.py

```diff
@@ -512,17 +512,54 @@ class ReportingPeriodListView(PermissionMixin, ListView):
     """ Currently the home view that lists the completed and missing time
     periods """
     context_object_name = "incomplete_reporting_periods"
-    queryset = ReportingPeriod.objects.all()
+    queryset = ReportingPeriod.objects.all().order_by('-start_date')
     template_name = "hours/reporting_period_list.html"
     permission_classes = (IsAuthenticated, )
 
+    def disallowed_dates(self, date):
+        # If the end of the fiscal year is not on a weekend, then create
+        # a list of dates around (the buffer) the end of the fiscal year
+        # where a reporting period will not be automatically created. This
+        # prevents the automatic creation of a reporting period that has
+        # working week days that span two fiscal years.
+        fy_start_date = datetime.date(year=date.year, month=10, day=1)
+        if fy_start_date.weekday() < 5 and date.month < 10:
```

Comment:
If the end date of the prior period is November or December, then the fiscal year switch for the calendar year already occurred and we don't need to worry about generating disallowable dates.